README.md

```diff
@@ -1,48 +1,61 @@
 
-- Ease of use and simple automation of HTTP/API scanning, fully working in CLI with a yaml configuration, taking advantage of [ZAP](https://www.zaproxy.org/)
-- Ability to run automated DAST scanning to suit various users' needs via custom container images
+RapiDAST provides:
+
+- Simplified HTTP/API security scanning using [ZAP]
+- Command-line execution with yaml configuration, suitable for integration in CI/CD pipelines
+- Ability to run automated DAST scanning with pre-built or custom container images
 - HTML, JSON and XML report generation
 - Integration with reporting solutions such as [OWASP DefectDojo](https://owasp.org/www-project-defectdojo/)
 
-# Getting Started
+RapiDAST is used for testing applications, and should not be used on production systems.
 
-## Prerequisites
+## Quickstart
 
-- `python` >= 3.6.8 (3.7 for MacOS/Darwin)
-- `podman` >= 3.0.1
-    + required when you want to run scanners from their container images, rather than installing them to your host.
-- See `requirements.txt` for a list of required python libraries
+Quickly setup RapiDAST to scan a target application. See [Workflow](#workflow) for more information.
+
+1. Create a minimal config file for the target application, see [Configuration](#configuration) section for details
+2. Run RapiDAST with the config file, either in a container or from source code
 
 ### OS Support
 
-Linux and MacOS`*` are supported.
+Linux and MacOS are both supported, however running RapiDAST in a container is currently only supported on Linux. See [MacOS Configuration](#macos) section for more details.
 
-#### Note regarding MacOS and ZAP
+### Run in container (Linux only)
 
-RapiDAST supports executing ZAP on the MacOS host directly only.
+Run the pre-built [rapidast container image](https://quay.io/repository/redhatproductsecurity/rapidast), which includes scanners like [ZAP]. Not compatible with config files using `general.container.type` set to `podman`.
 
-To run RapiDAST on MacOS(See the Configuration section below for more details on configuration):
+**Prerequisites**
 
-* Set `general.container.type: "none"` or `scanners.zap.container.type: "none"` in the configuration.
-* Configure `scanners.zap.container.parameters.executable` to the installation path of the `zap.sh` command, because it is not available in the PATH. Usually, its path is `/Applications/ZAP.app/Contents/Java/zap.sh` on MacOS.
+- `docker` / `podman` (>= v3.0.1)
 
-Example:
-
-```yaml
-scanners:
-  zap:
-    container:
-      type: none
-      parameters:
-        executable: "/Applications/ZAP.app/Contents/Java/zap.sh"
+**Run**
+```sh
+$ podman run -v ./config.yaml:/opt/rapidast/config/config.yaml:Z quay.io/redhatproductsecurity/rapidast:latest ./rapidast.py
```

Comment:
/opt/rapidast -> /home/rapidast

Reply:
would it be nicer to add a mention that '/home/rapidast/config/config.yaml' is a default config file path and otherwise '--config <config_file_path>' can be used?

Reply:
/opt/rapidast is correct.

Reply:
> would it be nicer to add a mention that '/home/rapidast/config/config.yaml' is a default config file path and otherwise '--config <config_file_path>' can be used?

To try and keep the quickstart section minimal, I think a better place for this might be showing the default in the help text.